src/test/java/rx/ObservableTests.java

```diff
@@ -1138,4 +1142,53 @@ public void testSubscribingSubscriberAsObserverMaintainsSubscriptionChain() {
 
         subscriber.assertUnsubscribed();
     }
+    
+    @Test
+    public void testExtend() {
+        final TestSubscriber<Object> subscriber = new TestSubscriber<Object>();
+        final Object value = new Object();
+        Observable.just(value).x(new ConversionFunc<Object,Object>(){
+            @Override
+            public Object convert(OnSubscribe<Object> onSubscribe) {
+                onSubscribe.call(subscriber);
+                subscriber.assertNoErrors();
+                subscriber.assertCompleted();
+                subscriber.assertValue(value);
+                return subscriber.getOnNextEvents().get(0);
+            }});
+    }
+
+    @Test
+    public void testOperatorConversion() {
+        final TestSubscriber<Object> subscriber = new TestSubscriber<Object>();
+        final Object value = new Object();
+        Observable.just(value).x(new OperatorConversion<Object,Object>(new Operator<Object, Object>(){
```

Comment:
This can be achieved with lifting before or in the `to()` operator:
```java
Observable.just(value).to(o -> o.lift(new Operator<Object, Object>() { ... }))
.subscribe(subscriber);
});
```